crates/store/re_chunk_store/src/dataframe.rs

```diff
@@ -296,12 +310,19 @@ impl ComponentColumnDescriptor {
 
     #[inline]
     pub fn to_arrow_field(&self) -> Arrow2Field {
+        let entity_path = &self.entity_path;
+        let descriptor = ComponentDescriptor {
+            archetype_name: self.archetype_name,
+            archetype_field_name: self.archetype_field_name,
+            component_name: self.component_name,
+        };
+
         Arrow2Field::new(
-            format!(
-                "{}:{}",
-                self.entity_path,
-                self.component_name.short_name().to_owned()
-            ),
+            // NOTE: Uncomment this to expose fully-qualified names in the Dataframe APIs!
+            // I'm not doing that right now, to avoid breaking changes (and we need to talk about
+            // what the syntax for these fully-qualified paths need to look like first).
+            format!("{}:{}", entity_path, descriptor.component_name.short_name()),
+            // format!("{entity_path}@{}", descriptor.short_name()),
```

Comment:
As mentioned, I kept the existing column-path syntax around for now, to avoid breaking anything downstream. TBD.